spark/src/test/scala/org/apache/spark/sql/delta/DeltaTypeWideningSuite.scala

```diff
@@ -67,15 +67,22 @@ trait DeltaTypeWideningTestMixin extends SharedSparkSession {
   protected def enableTypeWidening(tablePath: String, enabled: Boolean = true): Unit =
     sql(s"ALTER TABLE delta.`$tablePath` " +
           s"SET TBLPROPERTIES('${DeltaConfigs.ENABLE_TYPE_WIDENING.key}' = '${enabled.toString}')")
+
+  protected def metadata(
```

Comment:
nit: method name too general. What is this metadata for?

Reply:
Updated + added a comment